docs/conf.py

```
@@ -12,9 +12,22 @@
+import django
+
+# import github_links
+
+sys.path.insert(0, str(Path("..").resolve()))
+
+os.environ["DJANGO_SETTINGS_MODULE"] = "config.settings.base"
+django.setup()
 
 project = "Care"
-copyright = """2023, Open Healthcare Network"""  # noqa: A001
+copyright = f"{date.today().year}, Open Healthcare Network"  # noqa: A001, DTZ011
```

Ruff (0.13.1)

Unused `noqa` directive (non-enabled: `A001`, `DTZ011`)

Remove unused `noqa` directive

(RUF100)